imod/mf6/ist.py

```diff
@@ -191,11 +194,11 @@ def __init__(
         mobile_immobile_mass_transfer_rate,
         decay=None,
         decay_sorbed=None,
-        bulk_density=None,
+        bulk_density: Optional[xr.DataArray] = None,
         distribution_coefficient=None,
```

Comment:
Shouldn't ``decay``, ``decay_sorbed``, ``distribution_coefficient`` also get a type annotation?

Reply:
They don't trigger a mypy error. Perhaps when we set the MyPy settings to a stricter profile they will. But at the moment i consider that out of scope